python/ray/util/spark/cluster_init.py

```diff
@@ -1275,6 +1362,10 @@ def setup_ray_cluster(
             use 0 instead.
             This argument is only available on spark cluster which spark driver node
             has GPUs.
+        heap_memory_worker_node: Optional[int]:
```

Comment:
Maybe just memory_worker_node to align with the `memory` resource name. You should also mention that setting this is basically setting `--memory` of `ray start` and actually doesn't limit the heap memory Ray can use.

Reply:
Make sense ! :)